.mdlrc

```diff
@@ -1,24 +1,8 @@
-# https://github.com/igorshubovych/markdownlint-cli
-# https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md
-# markdownlint -c .github/linters/.markdown-lint.yml .
-
+rules '~MD013', '~MD024', '~MD025', '~MD026', '~MD033', '~MD040', '~MD041'
```

Comment:
it is just a refactor of this file? I think that grouping rules+description as before may be clearer

Reply:
i've changed the pre-commit hook that checks this markdownlint, so it has different way to inform the rules